pom.xml

```diff
@@ -20,37 +20,23 @@
 		<skip.integration.tests>false</skip.integration.tests>
 		<skip.unit.tests>false</skip.unit.tests>
 
-		<java.version>11</java.version>
+		<java.version>21</java.version>
 
-		<avro.version>1.8.1</avro.version>
-		<ch-kafka.version>1.4.4</ch-kafka.version>
-		<kafka-models.version>1.0.25</kafka-models.version>
-		<structured-logging.version>1.9.14</structured-logging.version>
-		<commons-lang3.version>3.11</commons-lang3.version>
-		<private.sdk.version>2.0.98</private.sdk.version>
-		<log4j-bom.version>2.16.0</log4j-bom.version>
+		<structured-logging.version>3.0.20</structured-logging.version>
+		<commons-lang3.version>3.17.0</commons-lang3.version>
 
 		<!-- Test -->
-		<mockito-junit-jupiter.version>3.8.0</mockito-junit-jupiter.version>
-		<equalsverifier.version>3.1.9</equalsverifier.version>
-		<testcontainers.version>1.15.3</testcontainers.version>
-		<system-lambda.version>1.2.0</system-lambda.version>
-		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
-		<maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
-		<jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+		<maven-surefire-plugin.version>3.5.2</maven-surefire-plugin.version>
+		<maven-failsafe-plugin.version>3.5.2</maven-failsafe-plugin.version>
+		<jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
 
 		<!-- Spring -->
-		<spring-boot-dependencies.version>2.6.3</spring-boot-dependencies.version>
-		<spring-test.version>5.2.0.RELEASE</spring-test.version>
-		<spring-boot-maven-plugin.version>2.2.0.RELEASE</spring-boot-maven-plugin.version>
-		<spring-kafka.version>2.8.2</spring-kafka.version>
-
-		<!-- Maven -->
-		<maven.compiler.source>11</maven.compiler.source>
-		<maven.compiler.target>11</maven.compiler.target>
+		<spring-boot-dependencies.version>3.4.0</spring-boot-dependencies.version>
+		<spring-kafka.version>3.3.0</spring-kafka.version>
 
 		<!-- Sonar dependency -->
-		<sonar-maven-plugin.version>3.7.0.1746</sonar-maven-plugin.version>
+    <!--Not sure if some of these are needed? Leaving in case there is some config im not aware of-->
```

Comment:
Think this is used by the pipeline sonar scans

Reply:
Yeah thought so which is why I left it in, will see PR though if it works, can always remove comment before we merge master